_includes/about/about.html

```diff
@@ -12,13 +12,23 @@
     <div class="info transite transite-up sn-1">
       <div class="title">{{ content.title }}</div>
       {% for item in content.descriptions %}
-        <div class="description">{{ item }}</div>
+      <div class="description">{{ item }}</div>
       {% endfor %}
+      <div class="description">
```

Comment:
Will this help?
https://shopify.github.io/liquid/filters/escape/
In yes, no need to remove item from YAML file